test/e2e/tls_util_test.go

```diff
@@ -28,6 +28,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+//todo(camilamacedo86): remove the  //nolint:staticcheck comment when the  TestCtx be removed
+//todo(camilamacedo86): replace TestCtx with Context
+
```

Comment:
Do we need all of these `nolint:staticcheck` comments in this file if we're using the new name?